JitsiConferenceEventManager.js

```diff
@@ -391,52 +391,6 @@ JitsiConferenceEventManager.prototype.setupChatRoomListeners = function() {
         }
     });
 
-    chatRoom.addPresenceListener('devices', (data, from) => {
-        let isAudioAvailable = false;
-        let isVideoAvailable = false;
-
-        data.children.forEach(config => {
-            if (config.tagName === 'audio') {
-                isAudioAvailable = config.value === 'true';
-            }
-            if (config.tagName === 'video') {
-                isVideoAvailable = config.value === 'true';
-            }
-        });
-
-        let availableDevices;
-
-        if (conference.myUserId() === from) {
-            availableDevices = conference.availableDevices;
```

Comment:
What a strange practice of modifying the object in place like this. It's possible `conference.availableDevices` might not be referenced anymore. I'm also unsure exactly how it differs from RTCUtils's available devices.

Reply:
Good catch! I'll chase it down and remove it if we no longer need it.